x-pack/plugins/maps/public/connected_components/mb_map/draw_control/draw_tooltip.tsx

```diff
@@ -81,6 +109,7 @@ export class DrawTooltip extends Component<Props, State> {
 
     return (
       <EuiPopover
+        panelPaddingSize="s"
```

Comment:
This padding is causing problems with the arrow as its not centered any more.
<img width="517" alt="Screen Shot 2021-06-15 at 1 43 30 PM" src="https://user-images.githubusercontent.com/373691/122115600-e21fc280-cde1-11eb-8a72-481bc0ed8e64.png">

Reply:
Removed. This is old from when we wanted the tooltip showing the index name being modified.